### ISSUE TITLE
ProvisioningEngine page shows flash "You are already signed in.", although no login is needed/supported for the simulator.

### ISSUE THREAD

**Issue by oveits** [oveits/ProvisioningEngine#43]
On Heroku, I have seen a flash "You are already signed in." on the upper right corner:
![image](https://cloud.githubusercontent.com/assets/8299115/13426343/14396a7e-dfad-11e5-9984-79d898039420.png)

For the ProvisioningEngine simulator, a login is not needed and not supported. Where does this flash come from? From Heroku? Or from ActiveAdmin? This is low prio, and I am adding it to the issue list as a reference only.

**Comment by oveits** [comment]
When entering https://provisioningengine.herokuapp.com/admin in the brower, I can see that I am logged into ActiveAdmin already. So, it is likely not a Heroku topic, but a side-effect of ActiveAdmin.

**Comment by oveits** [comment]
After refreshing the original link https://provisioningengine.herokuapp.com/customers, the flash is gone. This is expected: flash messages appear only once after being defined. They disappear when refreshing a page or switching to another page.

**Comment by oveits** [comment]
I won't fix this minor bug, so I will close it.
